inference-engine/tests/unit/vpu/middleend_tests/edges_tests/stage_dependency_edge.cpp

```diff
@@ -403,4 +403,37 @@ TEST_F(StageDependencyEdgeProcessingTests, RemoveStageDependencyUpdatesNextPrevS
     ASSERT_EQ(it, nextStages.end());
 }
 
-} // namespace vpu
\ No newline at end of file
+TEST_F(StageDependencyEdgeProcessingTests, RemoveStageDependencyViaDataToShapeEdgeUpdatesNextPrevStages) {
```

Comment:
I think more tests should be added in the future

Reply:
#-34798